.env.example

```diff
@@ -1,15 +1,26 @@
 # GENERAL
 
-## The network used for testing purposes
+## The network used
 NETWORK_NAME="sepolia" #  ["mainnet", "sepolia", "polygon", "polygonMumbai"]
 
-# PROTOCOL CONTRACTS
+## DEPLOYMENT
+DEPLOYMENT_RPC_ENDPOINT="https://host/path"
 
+## Hex encoded private keys separated by a comma `,`
+## The first item will be the deployment wallet
+PRIVATE_KEY="0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
```

Comment:
Hope this is not the actual private key. Better to remove and have "0x..."

Reply:
This value is the default key that HardHat uses on their template. I left it here since this already works